mindbender/maya/pythonpath/userSetup.py

```diff
@@ -2,6 +2,50 @@
 
 from maya import cmds
 
+import os
+
+
+def mayafpsconverter(Sfps):
```

Comment:
Yes, the second argument to `get` is the default value, if `fps` isn't available.

```python
assert {"hello": "world"}.get("notexist", "defaultvalue") == "defaultvalue"
```